.rubocop.yml

```diff
@@ -32,8 +37,11 @@ Layout/EmptyLines:
 Layout/ExtraSpacing:
   Severity: warning
 
-Layout/IndentationWidth:
+Layout/IndentationConsistency:
   Severity: warning
 
-Layout/IndentationConsistency:
+Layout/IndentationWidth:
   Severity: warning
+
+# Layout/LineLength:
+#   Enabled: false
```

Comment:
Are these extra comment lines necessary ?

Reply:
:shrug: I just fixed the alphabetical order -- assumed it is necessary since someone left it as a comment instead of removing it but we can remove it.

Reply:
Done